### PR TITLE
GH-929: Upload file to Wordpress component

### PR DESCRIPTION
**Pull request by tamarayustian**
It's finally here!!! Can upload image to our Wordpress, but need to update your .env to test this. I will upload new `/ui` env to our Discord in #database channel

### CODE REVIEW

File: ui/src/components/admin/announcements/AnnouncementEditorModal.js [hunk 1]
```diff
@@ -153,6 +155,7 @@ const AnnouncementEditorModal = (props) => {
       });
       return status;
     } else {
+      console.log(data.imageAdUrl);
```
Comment (albertparedandan) [inline ui/src/components/admin/announcements/AnnouncementEditorModal.js:158]: wtf is this

Comment (albertparedandan) [inline ui/src/components/admin/announcements/AnnouncementEditorModal.js:158]: remove pls

**Review by ksuhartono97 — COMMENTED** [review]
i have a CORS error so cannot verify if functionality works. not sure if anyone faces the same issue. if no one else has issue and our server will be ok then i think can merge

**Comment by tamarayustian** [comment]
- change the image link field to be a text field and put a button to upload file beside it

**Comment by tamarayustian** [comment]
image

updated the field to look like this as per suggestion

**Review by ksuhartono97 — APPROVED** [review]

**Pull request merged**